src/main/java/mekanism/common/block/BlockTransmitter.java

```diff
@@ -424,4 +442,24 @@ public TileEntity createNewTileEntity(World worldIn, int meta)
 		
 		return null;
 	}
+	
+	private Optional<TileEntitySidedPipe> getTileEntitySidedPipe(TileEntity entity)
```

Comment:
Is it necessary to return `Optional` instead of `null` like `getTileEntity` does?